app/views/submissions/show.html.erb

```diff
@@ -0,0 +1,11 @@
+<div>
+  <h2><%=@submission.title%></h2>
+  <h3><%=@submission.submitted_to%></h3>
```

Comment:
There are a few place like this, but there should be a whitespace after `=` and before closing `%`.